smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/StructureGenerator.java

```diff
@@ -168,8 +169,17 @@ private void renderErrorStructure() {
 
     private void renderStructureNamespace() {
         writer.addImport("isa", "__isa", "@aws-sdk/smithy-client");
+        writer.addImport("SENSITIVE_STRING", "SENSITIVE_STRING", "@aws-sdk/smithy-client");
         Symbol symbol = symbolProvider.toSymbol(shape);
         writer.openBlock("export namespace $L {", "}", symbol.getName(), () -> {
+            writer.openBlock("export const filterSensitiveLog = (obj: $L): any => ({", "})",
+                symbol.getName(),
+                () -> {
+                    StructuredMemberWriter config = new StructuredMemberWriter(
```

Comment:
This should have a clearer name than `config`, as this isn't configuration.

Reply:
The name was picked up from existing code https://github.com/awslabs/smithy-typescript/blob/4d52bb7163e22d0143f0be2d30196ab4bd637556/smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/StructureGenerator.java#L101-L102

Updated it to `structuredMemberWriter` in both the cases in commit https://github.com/awslabs/smithy-typescript/pull/170/commits/81ff15cb93b90a598fd30a5b5536ef800eafb8c8